src/components/app/variablesV2.css

```diff
@@ -120,11 +116,11 @@ or "warn/action" ineastd of "red/green"
   --color-content-grayblue: #7383a7;
   --color-background-wrapper: var(--color-mystic);
   --color-background-header: #fbfcfd;
-  --color-background-menu: #f8f9fb;
+  --color-background-menu: var(--color-white-smoke);
```

Comment:
As discussed, we this variable is being used in other components so we should remove them later, since they are outside the scope of the project, and it's just to keep the same colors across the application